doc/index.html

```diff
@@ -0,0 +1,12 @@
+<!-- HTML index file used in https://tarantool.github.io/ -->
```

Comment:
Nit: I guess it'll be `https://tarantool.github.io/tarantool` (as I see from other repositories).